Multisig/UI/Transactions/TransactionsList/TransactionsViewModel.swift

```diff
@@ -13,6 +13,13 @@ class TransactionsViewModel: LoadableViewModel {
     @Published var transactionsList = TransactionsListViewModel()
     @Published var isLoading: Bool = true
     @Published var errorMessage: String? = nil
+    @Published var isLoadingNextPage: Bool = true
+
+    var viewState = App.shared.viewState
+
+    
+    private var prevURL: String?
```

Comment:
please use the full name for this variable that has the class scope: `previousURL`

Reply:
is the previous URL used anywhere? do we need it at all?

Reply:
It's returned from the backend but it's not used